jellyfin-api/src/commonMain/kotlin/org/jellyfin/sdk/api/client/ApiClient.kt

```diff
@@ -91,8 +91,16 @@ public abstract class ApiClient {
 		pathParameters: Map<String, Any?> = emptyMap(),
 		queryParameters: Map<String, Any?> = emptyMap(),
 		requestBody: Any? = null,
+		expectedResponseCodes: IntRange = 200 until 300
 	): RawResponse
 
+	public abstract suspend fun headRequest(
```

Comment:
The ApiClient should only have one request function, no additional one should be added just for HEAD requests.

Reply:
On review, I think this PR is a little heavy handed in general. I'm going to start over now that I understand the code better and use a lighter touch.